src/main/java/io/stargate/sgv2/jsonapi/service/resolver/DropIndexCommandResolver.java

```diff
@@ -16,8 +25,26 @@ public Class<DropIndexCommand> getCommandClass() {
   }
 
   @Override
-  public Operation resolveTableCommand(
-      CommandContext<TableSchemaObject> ctx, DropIndexCommand command) {
-    return new DropIndexOperation(ctx, command.indexName());
+  public Operation resolveKeyspaceCommand(
+      CommandContext<KeyspaceSchemaObject> ctx, DropIndexCommand command) {
+    final SchemaAttempt.SchemaRetryPolicy schemaRetryPolicy =
+        new SchemaAttempt.SchemaRetryPolicy(
+            2,
```

Comment:
If we are getting the retry from config we should also get the retry count from config